test/tests/device/isAndroidWebview.js

```diff
@@ -10,6 +10,20 @@ describe("isAndroidWebview", () => {
       throw new Error(`Expected true, got ${JSON.stringify(bool)}`);
     }
   });
+  it("should return true when isAndroid function returns true, wv is present in user agent, and isOperaMini function returns false", () => {
+    window.navigator.userAgent = "AndroidVersion/9wv";
+    const bool = isAndroidWebview();
+    if (!bool) {
+      throw new Error(`Expected true, got ${JSON.stringify(bool)}`);
```

Comment:
Is `JSON.stringify()` necessary here (same question for the other tests)? I think this should work:
```suggestion
      throw new Error(`Expected true, got ${bool}`);
```

Reply:
It was already there. I know that TypeScript would complain about this though.

Reply:
Ah, gotcha. All good then. 👍